### PR TITLE
Adding next steps in first project setup

### PR DESCRIPTION
**Pull request by mkiebele**

### CODE REVIEW

File: Documentation/FirstProject/NextStepsFirstProject.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+===========================
+
+To create a sitepackage, you have two main options:
+
+*   Manual creation: For full control over your project's setup, you can follow
+    the detailed instructions int the
+    :ref:`TYPO3 site package tutorial <t3sitepackage:start>`.
+
+*   Using the `Sitepackage-Builder <https://www.sitepackagebuilder.com/>`__:
+    If you're looking for a faster start,
```
Comment (linawolf) [inline Documentation/FirstProject/NextStepsFirstProject.rst:27]:
```suggestion
    If you are looking for a faster start,
```

File: Documentation/FirstProject/NextStepsFirstProject.rst [hunk 2]
```diff
@@ -0,0 +1,50 @@
+..  include:: /Includes.rst.txt
+
+..  _next-steps-first-project:
+
+==========
+Next steps
+==========
+
+The next step is to create a sitepackage. In TYPO3, a sitepackage is a
```
Comment (linawolf) [inline Documentation/FirstProject/NextStepsFirstProject.rst:9]:
```suggestion
The next step is to create a site package. In TYPO3, a sitepackage is a
```

File: Documentation/FirstProject/CreateSitePackage.rst [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+..  _how-to-create-a-site-package:
+
+How to create a site package
+============================
+
+To create a site package, you have two main options:
+
+*   Manual creation: For full control over your project's setup, you can follow
+    the detailed instructions int the
```
Comment (froemken) [inline Documentation/FirstProject/CreateSitePackage.rst:23]:
```suggestion
    the detailed instructions in the
```

File: Documentation/FirstProject/CreateSitePackage.rst [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    :ref:`TYPO3 site package tutorial <t3sitepackage:start>`.
+
+*   Using the `Sitepackage-Builder <https://www.sitepackagebuilder.com/>`__:
+    If you are looking for a faster start, the Sitepackage-Builder is a
+    handy tool. You can choose whether your site package should be based on
+    the `Bootstrap Package <https://www.bootstrap-package.com/>`__ or
+    :ref:`fluid_styled_content <typo3/cms-fluid-styled-content:introduction>`.
+
+
+.. _pros-and-cons-of-sitepackage-builder:
```
Comment (froemken) [inline Documentation/FirstProject/CreateSitePackage.rst:33]:
```suggestion
..  _pros-and-cons-of-sitepackage-builder:
```

File: Documentation/FirstProject/CreateSitePackage.rst [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+Pros and cons of using the Sitepackage-Builder
+==============================================
+
+Pros:
+
+*   The Sitepackage-Builder is particularly useful for beginners or projects
+    that need a quick start.
+Cons:
```
Comment (froemken) [inline Documentation/FirstProject/CreateSitePackage.rst:42]: Don't know, if it needs extra space here, but looks ugly to me.

```suggestion

Cons:
```

Comment (linawolf) [reply Documentation/FirstProject/CreateSitePackage.rst:42]: yes Extra spaces after lists is needed

**Review by linawolf — APPROVED** [review]

**Pull request merged**